### DISCUSSION TITLE
What is the second parameter on `actionResponse` on `todos` page

### DISCUSSION THREAD

**Discussion by aaronksaunders** [Hebilicious/form-actions-nuxt#60] [Q&A]
```
 return actionResponse(event, { user }, { redirect: "/todos" })
```

I know this is sending a response to redirect, but how does one access `user`? Not sure why that object is there... I cannot finding anywhere once I am on the `todos` page.

**Comment by Hebilicious** [comment]
@aaronksaunders This is an oversight and that object can't be accessed easily.

You would have to pass it around manually.

To access the data in another page, we could introduce a new composable. Maybe `fromRedirect` or `fromAction`.
But the cleanest option is probably to serialise that state and place it in the url.

I need to think about it a little more, but I'm open to suggestions. If you come from a Remix background, I'd appreciate your feedback.

PS : I'm thinking of making a breaking change such as `actionResponse(event, { data: { user }, redirect: "/todos" })`
or introduce a new composable : `actionRedirect(event, { url: "/todos" })` to encourage people to use a different pattern.
